OtripleS.Web.Api/Controllers/TeachersController.cs

```diff
@@ -22,6 +22,41 @@ public class TeachersController : RESTFulController
         private readonly ITeacherService teacherService;
 
         public TeachersController(ITeacherService teacherService) =>
-            this.teacherService = teacherService;        
+            this.teacherService = teacherService;
+
+        public async ValueTask<ActionResult<Teacher>> GetTeacherAsync(Guid teacherId)
```

Comment:
HttpGet attribute is missing. Let us add it on GetById 

It should be [HttpGet("{teacherId}")]

Reply:
@viralpandya  thank you, [HttpGet("{teacherId}")] added.